test/utils/UnitTestClock.scala

```diff
@@ -16,7 +16,7 @@
 
 package utils
 
-import org.joda.time.{DateTime, DateTimeZone}
+import java.time.{Clock, Instant, LocalTime, ZoneId, ZonedDateTime}
 
 object UnitTestClock extends Clock {
```

Comment:
I don't thnk we even need this file at all? I think we can just replace all usages of the `.now()` method and inline `ZonedDateTime.now()`